plugins/workspace-plugin/src/workspace-projects-manager.ts

```diff
@@ -130,44 +135,42 @@ export class WorkspaceProjectsManager {
     );
   }
 
-  async updateOrCreateProjectInWorkspace(projectFolderURI: string | undefined): Promise<void> {
-    if (!projectFolderURI) {
-      return;
+  async watchMultiRootProjects(): Promise<void> {
+    if (fs.existsSync(this.projectsRoot)) {
```

Comment:
do we have a reason to use `existsSync()` instead of `pathExists()`?

Reply:
Changed on `await fs.pathExists(this.projectsRoot)`